backend/src/test/java/gov/cdc/usds/simplereport/service/TestOrderServiceTest.java

```diff
@@ -82,4 +89,24 @@ public void editTestResult() {
         assertEquals(TestResult.POSITIVE, queue.get(0).getTestResult());
         assertEquals(devA.getInternalId(), queue.get(0).getDeviceType().getInternalId());
     }
+
+    @Test
+    public void fetchTestResults() {
+        Organization org = _organizationService.getCurrentOrganization();
+        Facility facility = _organizationService.getFacilities(org).get(0);
+        Person p = _dataFactory.createFullPerson(org);
+        TestEvent _e = _dataFactory.createTestEvent(p, facility);
+
+        // Count queries with one order
+        hibernateQueryInterceptor.startQueryCount();
+        _service.getTestResults(facility.getInternalId().toString());
+        assertEquals(8, hibernateQueryInterceptor.getQueryCount());
+
+        // Count queries with three order
+        TestEvent _e1 = _dataFactory.createTestEvent(p, facility);
+        TestEvent _e2 = _dataFactory.createTestEvent(p, facility);
+        hibernateQueryInterceptor.startQueryCount();
+        _service.getTestResults(facility.getInternalId().toString());
+        assertEquals(8, hibernateQueryInterceptor.getQueryCount());
```

Comment:
This seems a little generous, still—what-all are we querying? Current org, facility by ID, test events... then probably some Person and and ApiUser queries? If it's that, there may still be room for improvement.